conda.recipe/meta.yaml

```diff
@@ -34,6 +34,7 @@ requirements:
     - scikit-image
     - scikit-learn
     - scipy
+    - progressbar
```

Comment:
The progressbar doesn't have very pretty output when running this on the cluster.

I made it optional in the near field script. Maybe we should axe the dependency, and it will only be used if it is installed?